clusterdock/cli.py

```diff
@@ -56,20 +56,17 @@ def main():
     build_parser = action_subparsers.add_parser('build',
                                                 formatter_class=FORMATTER_CLASS,
                                                 add_help=False)
-    build_parser.add_argument('--namespace',
-                              help='Namespace to use when looking for images',
-                              metavar='ns')
+    build_parser.add_argument('--repository',
```

Comment:
Nit: can you put this after `--operating-system` to keep it alphabetically ordered?